src/main/java/de/hysky/skyblocker/skyblock/chat/chatcoords/ChatWaypointLocation.java

```diff
@@ -0,0 +1,70 @@
+package de.hysky.skyblocker.skyblock.chat.chatcoords;
+
+import de.hysky.skyblocker.config.SkyblockerConfigManager;
+import de.hysky.skyblocker.utils.Constants;
+import de.hysky.skyblocker.utils.Utils;
+import de.hysky.skyblocker.utils.waypoint.Waypoint;
+import net.fabricmc.fabric.api.client.message.v1.ClientReceiveMessageEvents;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.text.ClickEvent;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import net.minecraft.util.math.BlockPos;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ChatWaypointLocation extends Waypoint {
```

Comment:
This shouldn't extend waypoint.
```suggestion
public class ChatWaypointLocation {
```